site/src/comparison.rs

```diff
@@ -376,19 +381,27 @@ impl ComparisonSummary {
         if self.num_regressions > 1 {
             writeln!(
                 result,
-                "- Average relevant regression: {:.1}%",
+                "- Arithmetic mean of relevant regressions: {:.1}%",
                 self.average_regression()
             )
             .unwrap();
         }
         if self.num_improvements > 1 {
             writeln!(
                 result,
-                "- Average relevant improvement: {:.1}%",
+                "- Arithmetic mean of relevant improvements: {:.1}%",
                 self.average_improvement()
             )
             .unwrap();
         }
+        if self.num_improvements > 1 && self.num_regressions > 1 {
```

Comment:
```suggestion
        if (self.num_improvements + self.num_regressions) > 1 {
```

Reply:
With this change it would show the "all changes" section even if there were no regressions. Maybe something like this:
```rust
if self.num_improvements > 0 && self.num_regressions > 0
```
?

Reply:
Ok!